.github/workflows/release-deploy.yml

```diff
@@ -29,3 +35,16 @@ jobs:
           custom_tag: ${{ github.event.inputs.deploy-env }}-${{ github.event.inputs.deploy-tag-version }}
           tag_prefix: ''
           github_token: ${{ secrets.PAT }}
+
+  trigger-deploy-to-aws:
```

Comment:
Could you add `needs: create-release-tag` so that `trigger-deploy-to-aws` does not start if `create-release-tag `fails?

Reply:
done